model_transactions_test.go

```diff
@@ -743,7 +757,19 @@ func (ts *EmbeddedDBTestSuite) TestTransaction_Save() {
 		// add the IN transaction
 		transactionIn := newTransaction(testTx2Hex, append(tc.client.DefaultModelOptions(), New())...)
 		require.NotNil(t, transactionIn)
+		// TODO: move this to one place
```

Comment:
General comment regarding TODOs/questions left as comments:
First category of TODOs
I would segregate the TODOs into ones you didn't find answers to but are key to closing this shuffle. Then team members could help you investigate.
For example:
https://github.com/BuxOrg/bux/pull/450/files#diff-e28c902ea80bcbe696ef6a0172e5c5cf443846c446de2830367cd36e192da4b0R124

Second category of TODOs
The second category of TODOs are those that we will not currently resolve but do not block the pull request merge.
In this case, I would create a task in Jira that contains the appropriate subtasks associated with the specific problem.

I don't mind leaving the TODOs as they are:
// TODO: BUX-XXX check if we can ommit broadcasting here

Reply:
TODO you referred to is not mine - I just this function from one file to another. I removed all TODOs and questions added by me.

BTW. if you think this function should go back to its previous file let me know.